config.proto

```diff
@@ -35,7 +35,9 @@ message AgentConfig {
 // Reporting covers the options related to the mechanics for sending data to the
 // tracing server o collector.
 message Reporting {
-    // endpoint represents the endpoint for reporting the traces e.g. http://api.traceable.ai:9411/api/v2/spans
+    // endpoint represents the endpoint for reporting the traces
+    // For ZIPKIN reporter type use http://api.traceable.ai:9411/api/v2/spans
```

Comment:
these should be https if we're use the traceable.ai as an example.

Also, for OTLP, isn't it grpc - in which case do we need the http:// ?

Reply:
It worked for gRPC. See https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#configuration-options

Reply:
ah, that's cool, I hadn't realized they've normalized that.